lib/net/http/requests.rb

```diff
@@ -1,67 +1,318 @@
 # frozen_string_literal: false
-#
+
 # HTTP/1.1 methods --- RFC2616
-#
 
-# See Net::HTTPGenericRequest for attributes and methods.
-# See Net::HTTP for usage examples.
+# \Class for representing
+# {HTTP method GET}[https://en.wikipedia.org/w/index.php?title=Hypertext_Transfer_Protocol#GET_method]:
+#
+#   require 'net/http'
+#   uri = URI('https://jsonplaceholder.typicode.com')
```

Comment:
IMO we should should just use `example.com` for the URL since we don't use the server's response. `example.com` makes it more clear that the URL is just a dummy placeholder.